planner/core/exhaust_physical_plans.go

```diff
@@ -2227,20 +2227,73 @@ func (la *LogicalAggregation) getStreamAggs(prop *property.PhysicalProperty) []P
 	return streamAggs
 }
 
+// TODO: support more operators and distinct later
+func (la *LogicalAggregation) checkCanPushDownToMPP() bool {
+	for _, agg := range la.AggFuncs {
+		if agg.HasDistinct {
+			return false
+		}
+		if agg.Name != ast.AggFuncSum && agg.Name != ast.AggFuncMin && agg.Name != ast.AggFuncCount && agg.Name != ast.AggFuncMax && agg.Name != ast.AggFuncFirstRow && agg.Name != ast.AggFuncAvg {
```

Comment:
I'm curious that whether these checks can be moved to "CanExprsPushDown" or be packed into a function in a swich-case style.

Reply:
updated.